app/labor/profile_image.rb

```diff
@@ -4,6 +4,7 @@ class ProfileImage
 
   def initialize(resource)
     @resource = resource
+    resource&.profile_image_url
```

Comment:
This is a bit weird, as you're testing the `profile_image_url` method call, and then keeping the error that the issues refer to in the following line

Reply:
Sorry, I got it wrong. I didn't get what I was supposed to do. I get it now what you mean. I will try to fix it. I also thought in doing something like putting a where like this `.where.not(user_id: nil)`in this query of `chat_channel.rb`: 
`relation = active_memberships.includes(:user).select(:id, :user_id, :last_opened_at)` inside of the method `channel_users` avoiding nil objects to be passed. What do you think?

Reply:
I'd stick to fixing the bug itself, as ProfileImage is called in a lot of places around the code, it's better to just guard against nil as an argument :) That's probably material for a separate PR if you want to explore that as well